tasks/postcss.ts

```diff
@@ -43,33 +47,49 @@ export = function(grunt: IGrunt) {
 		})
 	];
 
-	const moduleFiles = [{
-		expand: true,
-		src: ['**/*.css', '!**/variables.css'],
-		exclude: '**/variables.css',
-		dest: distDirectory,
-		cwd: 'src'
-	}];
+	function moduleFiles(dest: string) {
+		return [{
+			expand: true,
+			src: ['**/*.css', '!**/variables.css'],
+			exclude: '**/variables.css',
+			dest: dest,
+			cwd: 'src'
+		}];
+	}
 
-	const variableFiles = [{
-		expand: true,
-		src: '**/variables.css',
-		dest: distDirectory,
-		cwd: 'src'
-	}];
+	function variableFiles(dest: string) {
+		return [{
+			expand: true,
+			src: '**/variables.css',
+			dest: dest,
+			cwd: 'src'
+		}];
+	}
 
 	grunt.config.set('postcss', {
 		options: {
 			map: true
 		},
-		modules: {
-			files: moduleFiles,
+		'modules-dev': {
+			files: moduleFiles(path.join(devDirectory, 'src')),
+			options: {
+				processors: moduleProcessors(devDirectory)
+			}
+		},
+		'variables-dev': {
```

Comment:
Thinking about it and I don't think we will need to ever run the variables task in dev mode so this part can be reverted.